lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss

```diff
@@ -27,6 +27,13 @@
         }
     }
 
+    .adf-textitem-default .adf-property-read-only {
+        .mat-form-field-label-wrapper {
```

Comment:
the problem was introduced with this change : https://github.com/Alfresco/alfresco-ng2-components/blob/e794d7c9a6a469b4d18ff601dda55f3ebb6eb0f3/lib/core/src/lib/card-view/components/card-view/card-view.component.scss#L74-L77 so we might want to fix it at the root instead of ping ponging the issue